src/pages/settings/OrganizationInformations.tsx

```diff
@@ -0,0 +1,350 @@
+import { useRef } from 'react'
+import styled from 'styled-components'
+import { gql } from '@apollo/client'
+
+import { useInternationalization } from '~/hooks/core/useInternationalization'
+import { Avatar, Button, Skeleton, Typography } from '~/components/designSystem'
+import { GenericPlaceholder } from '~/components/GenericPlaceholder'
+import EmojiError from '~/public/images/exploding-head.png'
+import { theme } from '~/styles'
+import {
+  useGetOrganizationInformationsQuery,
+  EditOrganizationInformationsDialogFragmentDoc,
+} from '~/generated/graphql'
+import {
+  EditOrganizationInformationsDialog,
+  EditOrganizationInformationsDialogRef,
+} from '~/components/settings/EditOrganizationInformationsDialog'
+import CountryCodes from '~/public/countryCode.json'
+
+gql`
+  fragment OrganizationInformations on Organization {
+    id
+    logoUrl
+    name
+    legalName
+    legalNumber
+    email
+    addressLine1
+    addressLine2
+    zipcode
+    city
+    state
+    country
+  }
+
+  query getOrganizationInformations {
+    currentUser {
+      organizations {
+        ...OrganizationInformations
+        ...EditOrganizationInformationsDialog
+      }
+    }
+  }
+  ${EditOrganizationInformationsDialogFragmentDoc}
+`
+
+const OrganizationInformations = () => {
+  const { translate } = useInternationalization()
+  const editDialogRef = useRef<EditOrganizationInformationsDialogRef>(null)
+  const { data, loading, error } = useGetOrganizationInformationsQuery()
+  const organization = (data?.currentUser?.organizations || [])[0]
+
+  if (!!error) {
```

Comment:
`if (!!error && !loading) {`

We want to display the loading state if loading

Reply:
Done